polkadot/node/core/approval-voting/src/lib.rs

```diff
@@ -643,32 +643,46 @@ impl CurrentlyCheckingSet {
 	}
 }
 
-async fn get_session_info<'a, Sender>(
+async fn get_extended_session_info<'a, Sender>(
 	runtime_info: &'a mut RuntimeInfo,
 	sender: &mut Sender,
 	relay_parent: Hash,
 	session_index: SessionIndex,
-) -> Option<&'a SessionInfo>
+) -> Option<&'a ExtendedSessionInfo>
 where
 	Sender: SubsystemSender<RuntimeApiMessage>,
 {
 	match runtime_info
 		.get_session_info_by_index(sender, relay_parent, session_index)
 		.await
 	{
-		Ok(extended_info) => Some(&extended_info.session_info),
+		Ok(extended_info) => Some(&extended_info),
 		Err(_) => {
 			gum::debug!(
 				target: LOG_TARGET,
 				session = session_index,
 				?relay_parent,
-				"Can't obtain SessionInfo"
```

Comment:
This is really not how the usage was intended, but that was the case before already.